media/css/firefox/ios.less

```diff
@@ -0,0 +1,690 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+// potential general styles for new mozilla id themed pages
+@mozIDBlue: #0c99d5;
+
+.mozid {
+    color: #fff;
+
+    #outer-wrapper {
+        .aurora-bars();
+    }
+
+    #wrapper {
+        width: 100%;
+        overflow: hidden;
+        margin: 0;
+        padding: 0;
+    }
+
+    h1, h2, h3, h4, h5, h6, .huge, .large {
+        color: #fff;
+        text-shadow: none;
+    }
+}
+
+.container {
+    .border-box();
+    width: @widthDesktop;
+    margin: 0 auto;
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: @widthTablet;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        width: @widthMobile;
+    }
+
+    @media only screen and (min-width: @breakMobileLandscape) and (max-width: @breakTablet) {
+        width: @widthMobileLandscape;
+    }
+}
+
+#tabzilla {
+    position: absolute;
+    top: 0;
+    right: 0;
+}
+
+// Header logo
+.fx-logo {
+    float: left;
+
+    a {
+        display: block;
+        width: 110px;
+        height: 115px;
+        margin-top: @baseLine;
+        .image-replaced();
+        background: url(/media/img/firefox/family/nav-sprite7.png) 0 0 no-repeat;
```

Comment:
I'm not sure I follow. This is for the Firefox logo, and really it's only temporary because once FxiOS has a wide release we'll be adding the family nav to this page, which is also why I used the family sprite instead of adding a new logo image (though there are other logo images I could use instead). This logo is a bit of a hack until we can add the family nav.